### DISCUSSION TITLE
ImunifyAV thinks PLYR is malware?

### DISCUSSION THREAD

**Discussion by PeterKnightDigital** [sampotts/plyr#2817] [General]
A Plesk extension 'ImunifyAV' is reporting malware on my plyr-video directory. Does anyone else experience this?

> Error: ImunifyAV has detected malware on the website. Clean it up manually or upgrade to ImunifyAV+ for automatic malware cleanup.

When I inspect the page, there's really nothing odd about it

`<!DOCTYPE html>
<html lang="en" >
<head>
  <meta charset="UTF-8">
  <title>A Video</title>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
<link rel='stylesheet' href='https://unpkg.com/plyr@3/dist/plyr.css'>
<link rel="stylesheet" href="./style.css">

</head>
<body>

<div class="xcontainer">
	<div 
	class="plyr__video-embed" id="player" 
	playsinline 
	controls 
	data-plyr-provider="youtube" 
	data-plyr-embed-id="CQA_inGeS2c"
	data-poster="[my-JPG-URL]" 
	>
	<iframe
    src="[my-YT-URL]"
    allowfullscreen
    allowtransparency
    allow="autoplay"
  ></iframe>

	</div>

	
</div>

  <script src='https://cdn.polyfill.io/v2/polyfill.min.js?features=es6,Array.prototype.includes,CustomEvent,Object.entries,Object.values,URL'></script>
<script src='https://unpkg.com/plyr@3'></script><script  src="./script.js"></script>

</body>
</html>

`
Just a false alarm?

**Comment by sampotts** [comment]
It’s polyfill.io
https://www.theregister.com/2024/06/25/polyfillio_china_crisis/

I’ll update the docs.
